README.md

```diff
@@ -10,4 +10,239 @@ TODO: description
+
+#### `GET /api/medicalContacts?owner=USERNAME` - Get medical contacts by owner
+
+**Returns**
+
+- An array of medical contacts created by user with username `owner` in alphabetical order with respect to the last name of the contact
+
+**Throws**
+
+- `400` if `owner` is not given
+- `404` if `owner` is not a recognized username of any user
+
+#### `POST /api/medicalContacts` - Create a new medical contact
+
+**Body**
+
+- `active` _{string}_ - Whether the medical contact is in use or not
+- `title` _{string}_ - The medical contact's title
+- `first_name` _{string}_ - The medical contact's first name
+- `last_name` _{string}_ - The medical contact's last name
+- `hospital` _{string}_ - The medical contact's associated hospital
+- `specialty` _{string}_ - The medical contact's specialty
+- `phone_number` _{string}_ - The medical contact's phone number
+- `notes` _{string}_ - Notes about the medical contact
+
+**Returns**
+
+- A success message
+- A object with the created medical contact
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the first_name, last_name, or phone_number is empty or a stream of empty spaces
+
+#### `PATCH /api/medicalContacts/:medicalContactId?` - Edit the information of a medical contact
+
+**Body**
+
+- `active` _{string}_ - Whether the medical contact is in use or not
+- `title` _{string}_ - The medical contact's title
+- `first_name` _{string}_ - The medical contact's first name
+- `last_name` _{string}_ - The medical contact's last name
+- `hospital` _{string}_ - The medical contact's associated hospital
+- `specialty` _{string}_ - The medical contact's specialty
+- `phone_number` _{string}_ - The medical contact's phone number
+- `notes` _{string}_ - Notes about the medical contact
+
+**Returns**
+
+- A success message
+- A object with the edited medical contact
+
+**Throws**
+
+- `403` if the user is not logged in
+- `400` If the first_name, last_name, or phone_number is empty or a stream of empty spaces
+
+#### `DELETE /api/medicalContacts/:medicalContactId?` - Delete a medical contact
+
+**Returns**
+
+- A success message
+
+**Throws**
+
+- `403` if the user is not logged in
+- `403` if the user is not the owner of the medical contacts
+- `404` if the medicalContactId is invalid
+
+### Medication
+
+#### `GET /api/medications?owner=USERNAME` - Get medications by owner
+
+**Returns**
+
+- An array of medications created by user with username `owner` in alphabetical order with respect to the name of the medication
+
+**Throws**
+
+- `400` if `owner` is not given
+- `404` if `owner` is not a recognized username of any user
+
+#### `GET /api/medications?name=NAME` - Get medications by name
+
+**Returns**
+
+- An array of medications with name `NAME` in alphabetical order with respect to the name of the medication
+
+**Throws**
+
+- `400` if `NAME` is not given
+
+#### `POST /api/medications` - Create a new medication
+
+**Body**
+
+- `active` _{string}_ - Whether the medication is in use or not
```

Comment:
same as above -- this should just be set to true automatically in the backend and not part of the api